sdk/test-utils/recorder/README.md

```diff
@@ -1,18 +1,500 @@
+
+Sanitizers can be added in two different ways.
+
+1. Pass them as part of the `recorder.start({ envSetupForPlayback, sanitizerOptions })` call. Sanitizers are applied on the recordings in record mode before they are saved.
+2. Call `recorder.addSanitizers(sanitizerOptions, ["record", "playback"])`. This way, the same sanitizer would be applied in both record and playback modes.
+
+### How to playback
+
+Once you have recorded something, you can run your tests again with `TEST_MODE` set to `playback`.
+
+You'll notice how the tests succeed much faster. That's because the requests don't reach the service endpoints, the recorder assisted by the test-proxy tool will respond every request with their matching copy stored in the recordings.
+
+### Update existing recordings
+
+Once you have your recorded files, to update them after changing one of the tests, simply re-run the tests with `TEST_MODE` set to `record`. This will override previously existing files.
+
+> **Note:** If you rename the file of the test, or the name of the test, the
+> path of the recording will change. Make sure to delete the recordings
+> corresponding to the deleted tests. If at any point in time you lose your
+> recordings, don't worry. Running your tests with `TEST_MODE=record` will
+> re-generate them.
+
+### Skipping tests
+
+Writing live tests can take considerable time, specially since each time you want to check that everything works fine, you potentially need to run again every test. You can specify what test to run by following Mocha's approach of setting certain tests to `it.only`, and also to skip specific tests with `it.skip`.
+
+If you launch the `recorder` in record mode with some of these changes (and given that you activate the recorder on `beforeEach`), only the files that relate to the changed tests will be updated. Skipped tests won't update their recordings.
+
+This way, you can focus on fixing a specific set of tests with `.only`, then remove all the `.only` calls and trust that the playback will keep confirming that the unaffected tests are fine and green.
+
+You can also skip specific tests with the following.
+
+```js
+import { isLiveMode } from "@azure-tools/test-recorder";
+
+it("test-title", function () {
+  // isPlaybackMode() and isRecordMode() methods are also available from recorder.
+  if (!isLiveMode()) this.skip(); // This skips the test in record and playback modes
+  // Test goes here...
+  // ...
+});
+```
+
+### Supporting parallelism
+
+A common issue while running integration tests is that, sometimes two individuals or machines might try to run the same set of tests against the same resource.
+
+This is not directly related to the `@azure-tools/test-recorder` package, but if you're getting into issues because of concurrent conflicting requests, we strongly suggest using randomly generated strings as prefixes or suffixes for the resources you create.
```

Comment:
Might be worth linking to the section on `Recorder.variable` in this subsection since you'd need to use that if generating random strings etc.

Reply:
This was also a section I lifted from recorder v1.
You raise a great point. 💯